IFComp/lib/IFComp/Controller/Admin.pm

```diff
@@ -0,0 +1,49 @@
+package IFComp::Controller::Admin;
+use Moose;
+use namespace::autoclean;
+
+BEGIN { extends 'Catalyst::Controller'; }
+
+=head1 NAME
+
+IFComp::Controller::Admin
+
+=head1 DESCRIPTION
+
+A controller for administrative functions
+
+=head1 METHODS
+
+=cut
+
+
+=head2 index
+
+=cut
+
+sub index :Path :Args(0) {
```

Comment:
The access to this needs some locking down. Could you please check to see if the current user possess the 'votecounter' role? before allowing access to this action? (If they do not, just redirect them to /.

Note that role support was only added in early August, so you'll have to merge in the current state of the master branch into this one to use it.